auth/gcloud/aio/auth/session.py

```diff
@@ -201,7 +200,7 @@ def session(self) -> Session:
         # symbol ensures we match the base class's definition for static
         # analysis.
         async def post(self, url: str, headers: Dict[str, str],
-                       data: Optional[str] = None, timeout: int = 10,
+                       data: Optional[str] = None, timeout: Timeout = 10,
```

Comment:
It looks like you've only defined `Timeout` on L82 (eg. the `if not BUILD_GCLOUD_REST:` block), so it would not be available here. Since `requests` wouldn't support `aiohttp.Timeout` values anyway, having a `Timeout = float` definition in the `if BUILD_GCLOUD_REST:` might do the trick?

Reply:
Good catch, thank you. You can't redefine type aliases. I've set the timeout type for non-aio code to float directly. Check it now, please.